### DISCUSSION TITLE
Update script for Raspberry Pi

### DISCUSSION THREAD

**Discussion by Weidav** [mikebrady/shairport-sync#1566] [Show and tell]
Hey guys,
I just wrote an [Update Script for the Raspberry Pi](https://github.com/Weidav/shairport_update_script) so that updating Shairport Sync with Airplay2 becomes a lot easier.
I basically just stacked all the steps together from the official build guide needed to update it.
For now it just support Raspberry Pi OS, as I am using it and I think most people do.
Let me know if you find it use-full.
Cheers.

**Comment by mikebrady** [comment]
Many thanks!

**Comment by ghost** [comment]
Thanks 👍
